README.md

```diff
@@ -45,13 +45,14 @@ which meanwhile has been obsoleted by
 For details see [below](#cmp-features-and-openssl-versions)
 or the HISTORY section of [`cmpClient.pod`](doc/cmpClient.pod#HISTORY).
 
-A further use case of this software is to provide early access to all new CMP
+A further use case of this software was to provide early access to all new CMP
 features defined in [CMP Updates](https://www.rfc-editor.org/rfc/rfc9480) and
 the [Lightweight CMP Profile (LCMPP)](https://www.rfc-editor.org/rfc/rfc9483).
 To this end, the software can use in addition the intermediate CMP library
 [CMPforOpenSSL](https://github.com/mpeylo/cmpossl), called `libcmp` below.
-This is needed only as long as special new CMP(v3) features are required
-that are not covered by the OpenSSL version being used.
+This was needed only as long as special new CMP(v3) features are required
+that are not covered by the OpenSSL version being used.<br>
+As of October 2025, `libcmp` is no more actively maintained.
```

Comment:
as above.

Reply:
At this point, added the word "deprecated".